packages/babel-generator/src/generators/flow.js

```diff
@@ -221,11 +221,7 @@ export function TypeAnnotation(node: Object) {
 }
 
 export function TypeParameter(node: Object) {
-  if (node.variance === "plus") {
-    this.token("+");
-  } else if (node.variance === "minus") {
-    this.token("-");
-  }
```

Comment:
We'll need to think through how to approach it. I just don't want to end up in a state where people get broken output or thrown exceptions.